WtWSMS/common/culture/names_equivalency/00_names.txt

```diff
@@ -2183,7 +2148,6 @@ jemima_female = { Jemima Yemima }
 jengo_male = { Jengo }
 jeremiah_male = { Geremia Ieremias Jeremiah }
 jeremy_male = { Germias Jeremy }
-jerome_male = { Gierominu Herome_nu Jerome }
```

Comment:
Same here?

Reply:
> Same here?

Yes, all forms of Jerome are now in hieronymos_male.